Source/WebCore/dom/Element.cpp

```diff
@@ -5488,7 +5488,7 @@ bool Element::checkVisibility(const CheckVisibilityOptions& options)
     auto* parent = parentElementInComposedTree();
     ASSERT(!parent || parent->computedStyle());
     return style->skippedContentReason().value_or(ContentVisibility::Visible) != ContentVisibility::Hidden
```

Comment:
I find these complex boolean combinations much easier to read if written as lambdas.